src/docfx/template/RazorTemplate.cs

```diff
@@ -56,6 +56,7 @@ private static TestServer StartServer()
             void ConfigureServices(IServiceCollection services)
             {
                 services.AddMvc()
+                        .AddRazorOptions(options => options.AllowRecompilingViewsOnFileChange = false)
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.razor.razorviewengineoptions.allowrecompilingviewsonfilechange?view=aspnetcore-2.2
The default value is true if the version is Version_2_1 or earlier. If the version is later and EnvironmentName is Development, the default value is true. Otherwise, the default value is false.

I saw our version is 2.2. Is `EnvironmentName` set to `Development` during the CI run?

Reply:
There is no default value set for `EnvironmentName` and thus ASPNET core assume it is development, setting `EnvironmentName` to production sounds like a better option